src/components/split-button/split-button.style.ts

```diff
@@ -5,18 +5,48 @@ import baseTheme from "../../style/themes/base";
 
 const StyledSplitButton = styled.div`
   ${margin}
-
   display: inline-block;
   position: relative;
 
+  ${StyledButton}:first-of-type {
+    border-bottom-right-radius: var(--borderRadius000);
+    border-top-right-radius: var(--borderRadius000);
+    border-bottom-left-radius: var(--borderRadius400);
+    border-top-left-radius: var(--borderRadius400);
+  }
+
+  ${StyledButton} {
+    &:hover {
+      border-bottom-right-radius: var(--borderRadius000);
+      border-top-right-radius: var(--borderRadius000);
+      border-bottom-left-radius: var(--borderRadius400);
+      border-top-left-radius: var(--borderRadius400);
+    }
+  }
+
+  a:hover {
+    border-radius: 0px;
+  }
+
   & > ${StyledButton} {
     margin: 0;
-
     &:focus {
       border: 3px solid var(--colorsSemanticFocus500);
       outline: none;
+      border-bottom-right-radius: var(--borderRadius400);
```

Comment:
You have these inverted

![image](https://user-images.githubusercontent.com/44157880/228322431-d44ec4cd-647b-478b-a3f7-3cf9dab89ba2.png)